src/bin/hq.rs

```diff
@@ -283,7 +299,13 @@ async fn command_worker_stop(
     opts: WorkerStopOpts,
 ) -> anyhow::Result<()> {
     let mut connection = get_client_connection(&gsettings.server_directory()).await?;
-    stop_worker(&mut connection, opts.worker_id)
+
+    let selector: WorkerSelector = match opts.selector {
+        WorkerSelectorArg::Id(job_id) => WorkerSelector::Specific(vec![job_id]),
+        WorkerSelectorArg::All => WorkerSelector::All,
+    };
+
+    stop_worker(&mut connection, selector)
         .await
         .map_err(|e| e.into())
```

Comment:
`.await?` should be the same as `.await.map_err(|e| e.into())`. `?` call `into`.